### DISCUSSION TITLE
enquiry

### DISCUSSION THREAD

**Discussion by jesutobi** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#46] [Outreachy 2022] [answered]
Good evening @BatoolMM i just realised that i didnt add my name to the google doc all this while.i have been seeing issues of outreachy contributors creating sub pages to OSCSA website all this while,and i didnt understand until now that i had to create a subpage of one of these task in the image below .now i am confused because the google doc sheet says its " no longer accepting pr on this task" 
![ot](https://user-images.githubusercontent.com/43034407/196533074-6193fec3-262f-435d-9c72-48aeb45c25c5.png)

**Comment by BatoolMM** [comment, marked as answer]
Hi @jesutobi, yes, this task is completed already!
